spec/features/draws/draw_start_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Draw activation' do
+  let(:draw) { FactoryGirl.create(:draw, status: 'draft') }
+
+  context 'as admin' do
+    before { log_in(FactoryGirl.create(:admin)) }
+
+    it 'can be initiated' do
+      create_draw_resources(draw)
+      visit draw_path(draw)
+      click_link('Begin draw process')
+      expect(page).to have_css('.flash-notice',
+                               text: 'Draw successfully initiated.')
+    end
+
+    it 'redirects to show on failure' do
+      visit draw_path(draw)
+      click_link('Begin draw process')
+      # TODO: figure out if there's a better assertion?
+      expect(page).to have_css('.flash-error')
+    end
+
+    def create_draw_resources(draw)
+      FactoryGirl.create_list(:student, 2, draw: draw)
+      draw.suites << FactoryGirl.create(:suite_with_rooms, rooms_count: 2)
+    end
```

Comment:
awesome! I still wanted to use an invalid draw for one spec, but I think it's ok as is.